R/compute-rd.R

```diff
@@ -1,15 +1,15 @@
 #' @title Compute results
 #'
 #' @description  This is a method for the [dplyr::compute()] generic.
-#' For a (lazy) duckplyr frame,
+#' For a (tethered) duckplyr frame,
 #' `compute()` executes a query but stores it in a (temporary) table,
 #' or in a Parquet or CSV file.
 #' The result is a duckplyr frame that can be used with subsequent dplyr verbs.
 #'
 #' @inheritParams dplyr::compute
-#' @param lazy Set to `TRUE` to return a lazy or `FALSE` to return an eager data frame,
-#'   see the "Eager and lazy" section in [duckdb_tibble()].
-#'   The default is to inherit the lazyness of the input.
+#' @param tether Set to `TRUE` to return a tether or `FALSE` to return an untethered data frame,
+#'   see the "Tethering" section in [duckdb_tibble()].
```

Comment:
instead, inherit it on the same manual page for a better reading experience.

Reply:
I don't follow.